### PR TITLE
change typography api (partially applied) + adjust CounterBadge for w…

### PR DESCRIPTION
**Pull request by jonathanadler**
IGNORE THIS PR!!!!!!
EXCEPT EGOZI!!!!!

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5888/

File: src/Text/Text.st.css [hunk 1]
```diff
@@ -22,10 +23,16 @@
...
+  --color-exposed: --overridable;
+}
+
+.root {
+  -st-extends: Typography;
   -st-states: disabled, secondary, light,
     skin(enum(standard, success, error, premium, disabled, primary)),
     size(enum(tiny, small, medium)), weight(enum(thin, normal, bold));
 
+  color: var(--color, var(--color-internal, value(THEME-TEXT-COLOR-PRIMARY)));
```
Comment (shlomitc) [inline src/Text/Text.st.css:35]:
```suggestion
  color: var(--color, var(--skin-color, value(THEME-TEXT-COLOR-PRIMARY)));
```

File: src/Text/Text.st.css [hunk 2]
```diff
@@ -22,10 +23,16 @@
 }
 
 .root {
+  --color-exposed: --overridable;
```
Comment (shlomitc) [inline src/Text/Text.st.css:26]: not needed

**Comment by shlomitc** [comment]
that's pretty much done, so we can close the POC as successful

**Pull request closed without merging**